src/adaptors/http2/http2_adaptor.c

```diff
@@ -3237,8 +3253,15 @@ void qd_http2_delete_connector(qd_dispatch_t *qd, qd_http_connector_t *connector
         //
         // Deleting a connector must delete the corresponding qdr_connection_t and qdr_http2_connection_t objects also.
         //
-        if (connector->ctx)
-            qdr_core_close_connection((qdr_connection_t  *)connector->ctx);
+        if (connector->ctx) {
+            qdr_connection_t  *conn = (qdr_connection_t  *)connector->ctx;
```

Comment:
this line is not according to clang-format, it is not aligned with the lane below

Reply:
We should make the clang-format as part of the build ASAP. We should just bite the bullet and do it. I will fix this. Thx.

Reply:
Maybe we can make `git clang-format` be part of precommit checks. Therefore, all new (added or modified) code will have to be formatted, while old code can remain unchanged.

Advantage: it should be reatively painless.

Reply:
I think we should do what you did earlier which is agree on a clang format, format *all* the code in the repo by running the tool and check all that modified code back into the repo. Then enable the tool in cmake so that it fails the build (locally and on CI) if the code violates the format. This is what I meant by bite the bullet and do it. :)

Reply:
I know. I proposed to do this in two steps, instead, and soften the bullet for the first step.

Reply:
@jiridanek 2 steps most definitely

Reply:
* https://github.com/skupperproject/skupper-router/issues/660
* https://github.com/skupperproject/skupper-router/issues/661